assets/scripts/components/free-text.js

```diff
@@ -0,0 +1,20 @@
+
+// make freetext headings equal height to align paragraph content
+function makeHeadingEqualHeight () {
```

Comment:
Based on your comment, it looks like you decided against using this js.  
Are you adding it to the codebase on the chance that we may need it? Would you like to provide screenshots to Nicola to make a decision on how it looks?